bark_ml/library_wrappers/lib_fqf_iqn_qrdqn/agent/fqf_agent.py

```diff
@@ -0,0 +1,234 @@
+import torch
+from torch.optim import Adam, RMSprop
+
+from bark_ml.library_wrappers.lib_fqf_iqn_qrdqn.model import FQF
+from bark_ml.library_wrappers.lib_fqf_iqn_qrdqn.utils import disable_gradients, update_params,\
+    calculate_quantile_huber_loss, evaluate_quantile_at_action
+from .base_agent import BaseAgent
+
+
+class FQFAgent(BaseAgent):
```

Comment:
These agents are not executable in BARK as they do not have a Plan(..) or Clone(..) function. See tf-agents library

Reply:
Updated the agents with Plan() and Clone() while making use of self.env._ml_behavior as env is specialization of SingleAgentRuntime() which defines _ml_behavior from blueprint provided during initialization. I shall shortly add tests for the agents which verify the plan method.